src/components/Card.tsx

```diff
@@ -4,20 +4,41 @@ import { useCollection } from 'hooks'
 import tw from 'twin.macro'
 import styled from 'styled-components'
 import { useTranslation } from 'react-i18next'
+import { motion } from 'framer-motion'
 
 type CardProps = {
   cardData: PokemonCard
 }
 
 const Card = ({ cardData }: CardProps) => {
+  const buttonTransition = {
+    duration: 0.5,
+  }
+  const buttonAnimation = {
+    scale: [0, 1],
+    borderRadius: ['10%', '30%', '10%'],
+  }
+
   const { t } = useTranslation()
   const { collection, dispatch } = useCollection()
   const [pathname, setPathName] = useState(window.location.pathname)
+  const [isHovered, setHovered] = useState(false)
 
   return (
     <CardContainer>
-      <Figure>
-        <FigImage src={cardData.images.small} />
+      <Figure
+        onMouseEnter={() => {
+          setHovered(true)
+        }}
+        onMouseLeave={() => {
+          setHovered(false)
+        }}
+      >
+        <FigImage
+          src={cardData.images.small}
+          transition={{ duration: 0.2, ease: 'easeOut' }}
```

Comment:
```suggestion
          transition={{ duration: 0.2, ease: 'easeIn' }}
```

You want the user to be able to see and click the underlining as soon as possible. eastOut is technically correct (fast at beginning and slow at end), but it's flipped here for some reason

Reply:
found the problem, I did not take out the transition ease in the twin macro styles, easeOut works as expected now